drivers/utilities/utilities.go

```diff
@@ -0,0 +1,193 @@
+			deleteCommands = append(deleteCommands, fmt.Sprintf("DELETE FROM %s WHERE `key`='%s'", tableName, currentCounter))
+		}
+
+	}
+
+	randomSqlCommands["insert"] = insertCommands
+	randomSqlCommands["select"] = selectCommands
+	randomSqlCommands["update"] = updateCommands
+	randomSqlCommands["delete"] = deleteCommands
+
+	// log.Infof("Insert Queries - [%v]", insertCommands)
+	// log.Infof("Select Queries - [%v]", selectCommands)
+	// log.Infof("Update Queries - [%v]", updateCommands)
+	// log.Infof("Delete Queries - [%v]", deleteCommands)
+
+	return randomSqlCommands
+
+}
+
+// GenerateRandomSQLCommands generates pairs of INSERT and SELECT queries for a database
+func GenerateSQLCommandPair(tableName string, appType string) map[string][]string {
+	var sqlCommandMap = make(map[string][]string)
+	var selectQuery string
+	randomKey := "key-" + RandomString(10)
+	randomValue := "value-" + RandomString(10)
+
+	insertQuery := fmt.Sprintf("INSERT INTO %s VALUES('%s', '%s')", tableName, randomKey, randomValue)
+	if appType == Postgres {
+		selectQuery = fmt.Sprintf("SELECT * FROM %s WHERE key='%s'", tableName, randomKey)
+	} else if appType == MySql {
+		selectQuery = fmt.Sprintf("SELECT * FROM %s WHERE `key`='%s'", tableName, randomKey)
+	}
+
+	sqlCommandMap["insert"] = append(sqlCommandMap["insert"], insertQuery)
+	sqlCommandMap["select"] = append(sqlCommandMap["select"], selectQuery)
+
+	return sqlCommandMap
+}
+
+// CreateHostNameForApp creates a hostname using service name and namespace
+func CreateHostNameForApp(serviceName string, nodePort int32, namespace string) (string, error) {
+	var hostname string
+
+	if nodePort != 0 {
+		k8sCore := core.Instance()
+		nodes, err := k8sCore.GetNodes()
+		if err != nil {
+			return "", err
+		}
+		hostname = nodes.Items[0].Name
+	} else {
+		hostname = fmt.Sprintf("%s.%s.svc.cluster.local", serviceName, namespace)
+	}
+
+	return hostname, nil
+}
+
+// ExtractConnectionInfo Extracts the connection information from the service yaml
+func ExtractConnectionInfo(ctx *scheduler.Context) (AppInfo, error) {
+
+	// TODO: This needs to be enhanced to support multiple application in one ctx
+	var appInfo AppInfo
+
+	for _, specObj := range ctx.App.SpecList {
+		if obj, ok := specObj.(*corev1.Service); ok {
+			appInfo.Namespace = obj.Namespace
+			// TODO: This needs to be fetched from spec once CloneAppContextAndTransformWithMappings is fixed
+			svc, err := core.Instance().GetService(obj.Name, obj.Namespace)
+			if err != nil {
+				return appInfo, err
+			}
+			nodePort := svc.Spec.Ports[0].NodePort
+			hostname, err := CreateHostNameForApp(obj.Name, nodePort, obj.Namespace)
+			if err != nil {
+				return appInfo, fmt.Errorf("Some error occurred while generating hostname")
+			}
+			appInfo.Hostname = hostname
+			appInfo.NodePort = int(nodePort)
+			if svcAnnotationValue, ok := obj.Annotations[svcAnnotationKey]; ok {
+				appInfo.StartDataSupport = (svcAnnotationValue == "true")
```

Comment:
No need to add parenthesis around `svcAnnotationValue == "true"`

Reply:
Done